src/app.js

```diff
@@ -455,10 +455,13 @@ var run = function () {
 
   function editorOnChange () {
     var input = editor.getValue()
+    // save contents
+    editor.setCacheFileContent(input)
```

Comment:
I would not add a save button. Save buttons always complicate things because you have to keep track of two different states of your data. In our case, this is even more complicated because the files are not really stored in files. People might expect the data turn up on their harddrive. We already have unlimited undo history. I think it would make sense to add an 'autocommit to git' in the future, but the localstorage should always reflect the current state.

Concerning this change itself: I think we had a save every 5 minutes. It might be worth checking whether hold-pressing a button is slower without such a delay.